cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/PushUtils.java

```diff
@@ -139,8 +141,24 @@ public static void pushPackages(String packageName, String sourceRoot, String in
         }
         
         if (installToRepo == null) {
-            // Get access token
-            String accessToken = checkAccessToken();
+            // The access token can be specified as an environment variable or in 'Settings.toml'. First we would
+            // check if the access token was specified as an environment variable. If not we would read it from
+            // 'Settings.toml'.
+
+            // Check if the access token is specified as an environment variable
+            String accessToken = System.getenv("BALLERINA_CENTRAL_ACCESS_TOKEN");
```

Comment:
It's better if can put "BALLERINA_CENTRAL_ACCESS_TOKEN" to a constant first.

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/14242/commits/051de5130e23039496be15fe8469f93bb3505e10